hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -148,33 +151,59 @@ class RobotHW : public InterfaceManager
     ERROR
   };
 
-  // Return (in realtime) the state of the last doSwitch()
+  /** \brief Return (in realtime) the state of the last doSwitch() */
```

Comment:
Again, the codebase isn't entirely consistent on this, but I think these `\brief`s (Here and `switchResult()`) should end in a period.